### PR TITLE
App Routing - Home Page

### LINKED ISSUE #3159: ♻️ App Router migration - Homepage
**Description**
Currently our Website is using Page Router. This results in a huge bundle size which results in a slower UX on the website.

**Tasks**
- [x] Migrate Homepage from Page Router to App Router

### PR DESCRIPTION
**Pull request by amankumarrr**
Fixed: #3159 
Affected Route: `/`

✅ Home page is served from App Routing
✅ Improved Cumulative Layout and Largest Content Paint 
✅ Cleaned up the unnecessary `divs` for the mobile view 
✅ Reduced Total blocking time by almost half 
✅ Page score went up for desktop 📈90+ from 70+

**Screenshot:**
image
**Figure: PageSpeedInsight report**

### CODE REVIEW

**Comment by amankumarrr** [comment]
Tested with live Banner and appears to be working fine.
image

**Comment by amankumarrr** [comment]
liveStreaming on mobile. The performance is lower and can be improved by refactoring the component 

image

**Comment by amankumarrr** [comment]
Created an issue for the remaining work on App Routing - https://github.com/SSWConsulting/SSW.Website/issues/3186

File: components/blocks/upcomingEvents.tsx [hunk 1]
```diff
@@ -1,46 +1,33 @@
...
-      today.setHours(0, 0, 0, 0);
-      const events =
-        data.events ||
-        (await client.queries.getFutureEventsQuery({
-          fromDate: today.toISOString(),
-          top: data.numberOfEvents,
-        }));
-      setLoading(false);
+  if (!events) {
+    return [];
```
Comment (Calinator444) [inline components/blocks/upcomingEvents.tsx]: I'd maybe add a check here to see if the page is in edit mode and populate the events using the graphql fetch if so.

I'm not seeing the events in the admin view on staging. (see docs for checking if Tina is in edit mode https://tina.io/docs/extending-tina/edit-state/)

image

File: app/live-steam-banner/live-stream.tsx [hunk 2]
```diff
@@ -104,20 +112,38 @@ export function LiveStream({ event, children }: LiveStreamProps) {
 
   return (
     <>
-      {(showBanner || params?.get("liveBanner")) && (
+      {showBanner ? (
```
Comment (Calinator444) [inline app/live-steam-banner/live-stream.tsx:115]: just wondering, why was the param fetch here removed? it's used to test the live stream banner when the event is coming up

File: components/blocks/upcomingEvents.tsx (hunk 1 above)
Comment (amankumarrr) [reply components/blocks/upcomingEvents.tsx]: Fixed the issue with the client side rendering of upcoming events. 
image

**Review by Calinator444 — APPROVED** [review]

File: app/live-steam-banner/live-stream.tsx (hunk 2 above)
Comment (amankumarrr) [reply app/live-steam-banner/live-stream.tsx:115]: As per our conversation, we are maintaining the logic for fetching parameters on the client side. Whenever there are parameters, they will be loaded upon request.

**Review by bradystroud — APPROVED** [review]

**Comment by bradystroud** [comment]
As per our conversation, we did lots of testing and everything seems to work on the home page and when editing with TinaCMS.

**Pull request merged**